app/views/layouts/application.html.erb

```diff
@@ -106,6 +106,15 @@
     <%= yield %>
   </div>
 </div>
+<% if Flipper.enabled?(:runtime_banner) %>
```

Comment:
non blocking: you can use the `FeatureFlag` object, which is a tiny wrapper over Flipper